server.js

```diff
@@ -6,6 +6,9 @@ const exphbs = require("express-handlebars");
 const helpers = require("./utils/helpers");
 const hbs = exphbs.create({ helpers });
 
+const { error } = require("console");
+const { json } = require("sequelize");
+
```

Comment:
nitpick: I think this got auto-added and you didn't notice.  You should remove this lines prior to merging.